pageserver/benches/bench_layer_map.rs

```diff
@@ -254,5 +254,34 @@ fn bench_from_real_project(c: &mut Criterion) {
     });
 }
 
-criterion_group!(benches, bench_from_captest_env, bench_from_real_project);
+// Benchmark using synthetic data. Arrange image layers on stacked diagonal lines.
+fn bench_sequential(c: &mut Criterion) {
+    let mut layer_map = LayerMap::default();
+    for i in 0..100_000 {
+        // % 100 => 17 sec (very bad, should be way under 1 sec. 20x room for improvement)
+        // % 1000 => 1.6 sec
+        // % 10_000 =>
+        // TODO try inserting a super-wide layer in between every 10
```

Comment:
I wonder why we are not reporting time of this initialization if it is so critical?

Reply:
We could. But this framework has a hard-coded minumum of 10 samples for any measurement. I don't want to run this 10 times just to measure the query performance.

So far I haven't been very happy with this framework.